design-toolkit/components/src/components/loading-placeholder/index.stories.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { LoadingPlaceholder } from '.';
+import { LoadingPlaceholderStylesDefaults } from './styles';
+
+const meta: Meta<typeof LoadingPlaceholder> = {
+  args: LoadingPlaceholderStylesDefaults,
+  argTypes: {
+    pulse: {
+      type: 'boolean',
+    },
+    shape: {
+      control: 'select',
+      options: ['circ', 'rect'],
+    },
+    theme: {
+      control: 'select',
+      options: ['dark', 'light'],
+    },
+  },
+  component: LoadingPlaceholder,
+  title: 'Components/LoadingPlaceholder',
+};
+
+export default meta;
+
+export const Default: StoryObj<typeof LoadingPlaceholder> = {
+  render: (args) => (
+    <div className='w-[280px]'>
+      <LoadingPlaceholder {...args} />
+    </div>
+  ),
+};
+
+export const Nested: StoryObj<typeof LoadingPlaceholder> = {
+  render: (args) => (
+    <div className='w-[280px]'>
+      <LoadingPlaceholder className='mb-xl p-m' {...args}>
+        <br />
+        <br />
+        <LoadingPlaceholder className='w-[75%]' {...args} />
```

Comment:
Can you explain why we'd have children inside the Skeleton? Or why we'd pass one inside of another?

Reply:
Allowing for nesting would make something like this trivial.

<img width="315" height="300" alt="image" src="https://github.com/user-attachments/assets/0eed1370-bc96-4b9a-a4d0-536bdfe6af9b" />


http://localhost:6006/?path=/story/components-loadingplaceholder--nested

Reply:
Alright, cool